assets/js/googlesitekit/datastore/site/consent-mode.js

```diff
@@ -168,8 +168,8 @@ const baseSelectors = {
 	},
 
 	/**
-	 * Returns true if Google Ads is in use, either through a linked Analytics & Ads account
-	 * or an Ads conversion tracking ID.
+	 * Returns true if Google Ads is in use, either through a linked Analytics & Ads
+	 * account, an Ads conversion tracking ID, or via Analytics tag config.
```

Comment:
It might be worth adding a `@since n.e.x.t` tag to track when this change was made.

Reply:
Added, thank you!